tests/release/release-failures.go

```diff
@@ -1,255 +0,0 @@
-package release
-
-import (
-	"time"
-
-	"github.com/google/uuid"
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
-	"k8s.io/apimachinery/pkg/api/meta"
-	klog "k8s.io/klog/v2"
-)
-
-const (
-	missingPipelineName    string = "missing-release-pipeline"
-	missingReleaseStrategy string = "missing-release-strategy"
-)
-
-var _ = framework.ReleaseSuiteDescribe("test-release-service-failures", Label("release"), func() {
```

Comment:
If you do not plan to remove these tests completely, you could just mark them as pending and they will be skipped during test execution.
```
var _ = framework.ReleaseSuiteDescribe("test-release-service-failures", Pending, Label("release"), func() {
```

It's better than removing/commenting out hundreds of lines of code and it's easier for a review (been there, done that :)

Reply:
Thanks Pavel for this important information